### ISSUE TITLE
Not working bug tracking after upate to 24.8.0.dev0 version

### ISSUE THREAD

**Issue by urec56** [getsentry/self-hosted#3228]
### Self-Hosted Version

24.8.0.dev0

### CPU Architecture

x86_64

### Docker Version

26.0.2

### Docker Compose Version

2.26.1

### Steps to Reproduce

1. I had a working sentry service.

2. I wanted to add email to the service so that I could invite people.

3. Changed sentry/config.yml according to the documentation

4. Run install.sh

5. After raising the service, my mail started working, but bug tracking stopped working and there was a constant 100% load on the CPU. In processes with high load, snuba, docker_entry, runc constantly appear. 

New Senry version is: 24.8.0.dev0

P.S. I changed the mail settings 3-4 times, so I restarted install.sh each time, if that matters.

### Expected Result

Working service with working SMTP 

### Actual Result

Working SMTP but not working service and constant high CPU load

[docker compose up -d.log](https://github.com/user-attachments/files/16379645/docker.compose.up.-d.log)
[ls -1 sentry_install_log-*.txt | tail -1 | xargs cat.log](https://github.com/user-attachments/files/16379646/ls.-1.sentry_install_log-.txt.tail.-1.xargs.cat.log)

![load screen](https://github.com/user-attachments/assets/77b50355-9582-4251-ac22-03ba73aa14fe)

![load screen](https://github.com/user-attachments/assets/d4e3a05b-89ac-4819-9632-3ae522708c60)

![load screen](https://github.com/user-attachments/assets/4b3b5418-3eac-4071-8766-2c03a404f66d)

**Comment by IanWoodard** [comment]
Which version are you upgrading from?
It looks like your issue may stem from not having enough cpu allocated. Could you try allocating more CPU to see if that solves the issue? The previous version you used may have required less resources to run.

**Comment by urec56** [comment]
I don't remember what version it was. Only thing I can say, that it was installed in May 2024

**Comment by urec56** [comment]
This version I installed with `SENTRY_IMAGE=getsentry/sentry:24.5.1 SNUBA_IMAGE=gets
entry/snuba:24.5.1 ./install.sh` without email.

![Image](https://github.com/user-attachments/assets/ebe63e9a-f34a-4a27-abca-de41e2eab783)

**Comment by urec56** [comment]
Reinstalled it with default `./install.sh`, no mail. Version now is `Sentry 24.8.0.dev0`.

![Image](https://github.com/user-attachments/assets/21ae39f7-ab8b-471c-8a52-bff3822f67b0)

No issues was tracked.

Tried new DSN link, no issues was tracked.

One detail that I didn't mention is that before adding the email, I added a new project - apple-ios, but no events have come to it yet. Maybe that's what's causing the problem.
